cmake/dependencies.cmake

```diff
@@ -1,8 +1,26 @@
-# >= 3.2 required for ExternalProject_Add_StepDependencies
-cmake_minimum_required(VERSION 3.2)
-
-
-include(${PROJECT_SOURCE_DIR}/lib/libappimage/cmake/scripts.cmake)
+# FetchContent_MakeAvailable() is only available in CMake 3.14 or newer
+cmake_minimum_required(VERSION 3.14)
+
+include(FetchContent)
+
+# Need this patch until https://github.com/AppImage/libappimage/pull/160 is resolved
+FetchContent_Declare(libappimage_patch
+    URL https://github.com/AppImage/libappimage/commit/b3398bb496e47947864b4b8bc2999c8427f86a9a.patch
+    DOWNLOAD_NO_EXTRACT TRUE
+)
+FetchContent_MakeAvailable(libappimage_patch)
+
+FetchContent_Declare(libappimage
+    # We can not use a URL source with a github-generated source archive: libappimage's gtest submodule would be missing
+    GIT_REPOSITORY https://github.com/AppImage/libappimage
+    GIT_TAG aa7d9fb03d3d64415c37120f20faa05412458e94  # latest as of 2022-08-18
+    # The patch command has || true to prevent the build from failing if the patch has already been applied
+    PATCH_COMMAND patch -p 1 < ${libappimage_patch_SOURCE_DIR}/b3398bb496e47947864b4b8bc2999c8427f86a9a.patch || true
```

Comment:
This will let unrelated errors slip. How about the `-N` flag? It should work with this patch.